src/UI/Images/transferFunctionManipulators.js

```diff
@@ -0,0 +1,177 @@
+import vtkMouseRangeManipulator from '@kitware/vtk.js/Interaction/Manipulators/MouseRangeManipulator'
```

Comment:
Next steps: To UI / Renderer independent manipulator that works with DOM events and renderer viewport state, etc. stored in the context, and send events to the machine.